content/composer/blueprint-creation.md

```diff
@@ -6,61 +6,67 @@ draft: false
+Each type will appear with the derived icon (from the parent type). 
 
 When a user logs in to the system he will see the list of built in stencils (types), as well as the
 stencils the user added in previous sessions.
 
-Stencils currently cannot be deleted – this functionality will be available in future releases.
+### Adding Nodes to The Blueprint
+Nodes are defined based on built in types, as well as added inline types. 
+Note that nodes are currently defined per cloud provider (i.e. OpenStack, CloudStack, etc), or as the basic Cloudify type nodes.
+When dragging nodes into the canvas areas, the composer will only allow performing legal actions. 
+This prohibits users from performing actions that are not consisting with topology concepts, for example hosting a volume inside a compute node, or defining a network within the volume.
+
+
+### Editing Nodes in The Blueprint
 
-# Adding Nodes to The Blueprint
-Nodes are defined based on built in types, as well as added inline types. Note that nodes are currently defined per cloud provider (i.e. OpenStack, CloudStack, etc), or as the basic Cloudify type nodes.
-When dragging nodes into the canvas areas, the composer will only allow performing legal actions. This prohibits users from performing actions that are not consisting with topology concepts, for example hosting a volume inside a compute node, or defining a network within the volume.
+Every node added to the canvas starts with the basic node implementation but in order to fully define the node it needs to be edited. 
+To edit the node simply click on it, and on the right side of the screen a window will open up with all configuration options for the node.
 
-# Deleting Nodes from The Blueprint
-Nodes can be deleted from the canvas area at any time by selecting the node and pressing the delete button on the details panel that open up. This operation cannot be reversed and requires a confirmation in the popup window that opens.
+
+#### Renaming a Node 
+By default, the node name would be its type followed by the next available index. 
+To change the node name press on it and replace with the text of choice. 
+Name should be unique within the blueprint.
+
+#### Deleting a Node 
+Nodes can be deleted from the canvas area at any time by selecting the node and pressing the delete button on the details panel that open up. 
+This operation cannot be reversed and requires a confirmation in the popup window that opens.
 Note that when deleting a node connected to others, the relationship connecting the deleted node will be deleted as well.
 
-# Editing Nodes in The Blueprint
-Every node added to the canvas starts with the basic node implementation but in order to fully define the node it needs to be edited. To edit the node simply click on it, and on the right side of the screen a window will open up with all configuration options for the node.
+#### Setting Number of Instances
+Unless otherwise stated, the number of instances for the node is 1. 
```

Comment:
"the number of instances for the node is 1" -> "the number of set node instances is 1"